recipes/rocksdb/all/patches/6.10.2/0001-Rename-jemalloc-according-to-conan-packages.patch

```diff
@@ -0,0 +1,14 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -101,9 +101,9 @@ else()
+     set(WITH_JEMALLOC ON)
+   else()
+     if(WITH_JEMALLOC)
+-      find_package(JeMalloc REQUIRED)
++      find_package(jemalloc REQUIRED)
+       add_definitions(-DROCKSDB_JEMALLOC -DJEMALLOC_NO_DEMANGLE)
+-      list(APPEND THIRDPARTY_LIBS JeMalloc::JeMalloc)
```

Comment:
I think we should fix jemalloc instead (adding `cpp_info.names`). otherwise, we'll have to patch every library using jemalloc...

Reply:
After a small search on github, it doesns't seems obvious if it should be spelled JeMalloc or Jemalloc. Anyway there is not naming from upstream neither from cmake...

Reply:
@danimtb @uilianries I can update jemalloc package name if required, what do you think about it ? Should I or not ?

Reply:
I think the correct name is **jemalloc**, all lower case: https://github.com/jemalloc/jemalloc-cmake/blob/master/CMakeLists.txt